UnityProject/Assets/Prefabs/UI/Tabs/Resources/TabPaper.prefab

```diff
@@ -135,8 +135,8 @@ MonoBehaviour:
   m_HideSoftKeyboard: 0
   m_CharacterValidation: 0
   m_RegexValue: 
-  m_GlobalPointSize: 26
-  m_CharacterLimit: 0
+  m_GlobalPointSize: 25
+  m_CharacterLimit: 1750
```

Comment:
This was for the guy who wanted to write long stuff in paper for there map

Reply:
yea, i added an option for more character and line limits with a warning on how to avoid blurry text